.github/workflows/update_protos.yml

```diff
@@ -8,12 +8,9 @@ on:
 
 jobs:
   update-protos:
-    if: github.repository_owner == 'viamrobotics'
```

Comment:
(q) why do we no longer want to check for `repository_owner == 'viamrobotics'`?

Reply:
Ooops that was an accident while I was testing. Added back in